pyproject.toml

```diff
@@ -22,10 +22,10 @@ classifiers = [
 "custom" = "resolver.custom:Custom"
 
 [tool.poetry.dependencies]
-python = "^3.8.1"
+python = ">=3.8,<3.11"
 
 [tool.poetry.group.dev.dependencies]
-pre-commit = "^3.2.1"
+pre-commit = "^3.0.0"
```

Comment:
had to lower version here otherwise we would get this error from the poetry-lock precommit linter.

```
Current Python version (3.11.2) is not allowed by the project (>=3.8,<=3.10.10).
Please change python executable via the "env use" command.
```